Tests/SonarQube.Common.UnitTests/AnalysisConfigTests.cs

```diff
@@ -59,12 +59,38 @@ public void AnalysisConfig_Serialization_SaveAndReload()
             originalConfig.ServerSettings = new AnalysisProperties();
             originalConfig.ServerSettings.Add(new Property() { Id = "server.key", Value = "server.value" });
 
+            originalConfig.AnalyzerSettings = new AnalyzerSettings();
+            originalConfig.AnalyzerSettings.RuleSetFilePath = "ruleset path";
+
+            originalConfig.AnalyzerSettings.AdditionalFilePaths = new List<string>();
+            originalConfig.AnalyzerSettings.AdditionalFilePaths.Add("additional path1");
+            originalConfig.AnalyzerSettings.AdditionalFilePaths.Add("additional path2");
+
+            originalConfig.AnalyzerSettings.AnalyzerAssemblyPaths = new List<string>();
+            originalConfig.AnalyzerSettings.AnalyzerAssemblyPaths.Add("analyzer path1");
+            originalConfig.AnalyzerSettings.AnalyzerAssemblyPaths.Add("analyzer path2");
```

Comment:
Updated the existing serialization test to check the analyzer settings are saved and reloaded.